Source/vendors/MathGeoLib/Math/Quat.cpp

```diff
@@ -537,14 +537,14 @@ void SetQuatFrom(Quat &q, const M &m)
 
 void Quat::Set(const float3x3 &m)
 {
-	assume(m.IsColOrthogonal());
-	assume(m.HasUnitaryScale());
-	assume(!m.HasNegativeScale());
+	// assume(m.IsColOrthogonal());
+	// assume(m.HasUnitaryScale());
+	// assume(!m.HasNegativeScale());
 	SetQuatFrom(*this, m);
 
 #ifdef MATH_ASSERT_CORRECTNESS
 	// Test that the conversion float3x3->Quat->float3x3 is correct.
-	mathassert(this->ToFloat3x3().Equals(m, 0.01f));
+	//mathassert(this->ToFloat3x3().Equals(m, 0.01f));
```

Comment:
We can revert these changes as in a prior commit we already disabled these asserts centrally from MathGeoLibConfig.h.

Reply:
this is also code that was merely for testing, thanks for the review